prover/protocol/distributed/compiler/permutation/permutation_test.go

```diff
@@ -129,7 +130,7 @@ func TestPermutation(t *testing.T) {
 
 					build.RegisterCommit(col.GetColID(), col.Size())
 				}
-			}, dummy.CompileAtProverLvl)
```

Comment:
It's not gonna work if you compile the GrandProduct query before declaring the GrandProduct query